www/jury/ajax_teams.php

```diff
@@ -5,7 +5,7 @@
 
 $teams = $DB->q("TABLE SELECT teamid AS id, name,
                  CONCAT(name, ' (t', teamid, ')') AS search FROM team
-                 WHERE (name COLLATE utf8_general_ci LIKE %c OR
-                 CONCAT('t', teamid) LIKE %c)", $_GET['q'], $_GET['q']);
+                 WHERE (name COLLATE %s_general_ci LIKE %c OR
+                 CONCAT('t', teamid) LIKE %c)", DJ_CHARACTER_SET_MYSQL, $_GET['q'], $_GET['q']);
```

Comment:
Indeed, and even if it works, since this is not user-provided content, there's no reason to escape it with '%s' operators from the libdb library. So I'd suggest as Thijs and simply string-join the new PHP constant.